### DISCUSSION TITLE
No data showing

### DISCUSSION THREAD

**Discussion by Carsonmiles1** [SauceLLC/sauce4strava#147] [General]
When opening the analyze option on a ride I get a blank page with no data showing, I also don't see the peaks on the left of the screen. Does anyone know a fix for this? When I remove Sauce the data comes back. I also have my Sauce linked to my Patreon. 
<img width="1169" alt="Screen Shot 2023-02-03 at 3 20 43 PM" src="https://user-images.githubusercontent.com/124372968/216626544-c016c03d-e674-40cc-8a11-9d7d1ead7ae5.png">

**Comment by mayfield** [comment]
Hi @Carsonmiles1 can you hit F12 and report what you see in the Console tab when you load that page?

**Reply by Carsonmiles1** [reply]
3VM154:2091 Sauce inteceptCallback failed (falling back to ajax): DOMException: Internal error.
(anonymous) @ VM154:2091
VM154:1307 Sauce Error: Error
    at Object.reportError [as error] (base.js:1294:32)
    at SyncManager.refreshLoop (chrome-extension://eigiefcapdcdmncdghkeahgfmnobigha/src/bg/hist.mjs:1981:30)
    at Object.getAthlete (proxy.js:123:27)
    at initSyncActivity (analysis.js:3243:46)
    at prepare (analysis.js:3298:33)
    at async Object.load (analysis.js:3579:13)
    at async analysis.js:3830:9
reportError @ VM154:1307
a.tiles.mapbox.com/v4/mapbox.satellite,mapbox.mapbox-terrain-dem-v1,mapbox.mapbox-streets-v8.json?access_token=pk.eyJ1Ijoic3RyYXZhIiwiYSI6IlpoeXU2U0UifQ.c7yhlZevNRFCqHYm6G6Cyg&secure:1          Failed to load resource: the server responded with a status of 400 (Bad Request)
manifest-6dc1a83d91fb449e38746b0546722f9661843a0a483b0af17f8f90ba25f9ae79.js:7 could not load TileJSON at mapbox://styles/strava/ckzn9habn001i15qtww7535k9
log @ manifest-6dc1a83d91fb449e38746b0546722f9661843a0a483b0af17f8f90ba25f9ae79.js:7
analysis.js:3836 Uncaught (in promise) Error
    at Object.reportError [as error] (base.js:1294:32)
    at SyncManager.refreshLoop (chrome-extension://eigiefcapdcdmncdghkeahgfmnobigha/src/bg/hist.mjs:1981:30)
    at Object.getAthlete (proxy.js:123:27)
    at initSyncActivity (analysis.js:3243:46)
    at prepare (analysis.js:3298:33)
    at async Object.load (analysis.js:3579:13)
    at async analysis.js:3830:9
a.tiles.mapbox.com/v4/mapbox.satellite,mapbox.mapbox-terrain-dem-v1,mapbox.mapbox-streets-v8.json?access_token=pk.eyJ1Ijoic3RyYXZhIiwiYSI6IlpoeXU2U0UifQ.c7yhlZevNRFCqHYm6G6Cyg&secure:1          Failed to load resource: the server responded with a status of 400 (Bad Request)
manifest-6dc1a83d91fb449e38746b0546722f9661843a0a483b0af17f8f90ba25f9ae79.js:7 could not load TileJSON at mapbox://styles/strava/ckzn9habn001i15qtww7535k9
log @ manifest-6dc1a83d91fb449e38746b0546722f9661843a0a483b0af17f8f90ba25f9ae79.js:7
4VM154:1184 Uncaught (in promise) DOMException: Internal error.
DevTools failed to load source map: Could not load content for chrome-extension://bmnlcjabgnpnenekpadlanbbkooimhnj/browser-polyfill.js.map: System error: net::ERR_BLOCKED_BY_CLIENT
DevTools failed to load source map: Could not load content for https://www.strava.com/activities/8497915925/load-image.all.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://www.strava.com/activities/8497915925/autotrack.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://www.strava.com/activities/8497915925/dist/dexie.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE

**Reply by Carsonmiles1** [reply]
<img width="546" alt="Screen Shot 2023-02-03 at 10 20 05 PM" src="https://user-images.githubusercontent.com/124372968/216712805-476476ae-8029-475e-8e75-98eb03969926.png">
hopefully this helps @mayfield

**Comment by mayfield** [comment]
Looks like something is wedged in the background sync engine.  Can you go to the URL chrome://extensions and click the "Developer mode" toggle on the top right.  This will show some more options for the extension and then you can click on "Inspect views: pages/bg.html".  It looks sort of like this...
![image](https://user-images.githubusercontent.com/139316/216723285-183d9f7f-4d08-4b0f-980b-d25c6464093a.png)

With that up can you send me what you see in the Console for it?  Once you've captured that I might be able to figure out what happened but we should try and get you back to working too, so back on the Extensions page try clicking the little reload button for Sauce and then reload your analysis page and see if you're back up.  If that doesn't work let's try an old fashion browser restart just to make sure there aren't any gremlins lurking.

**Comment by Carsonmiles1** [comment]
Here's what I see,
<img width="641" alt="Screen Shot 2023-02-06 at 8 28 58 PM" src="https://user-images.githubusercontent.com/124372968/217067883-280eee70-4e72-4a81-a8a9-5c0711430c1f.png">

Updating my chrome and restarting the browser has seemed to fix the problem on my end! Thanks for your help.

**Comment by mayfield** [comment]
Glad the restart fixed it.  That `Internal error` is most likely from IndexedDB.  On very rare occasion when the browser's IndexedDB implementation breaks I've seen this.  Luckily I've never seen it lead to data loss, but make an export of your activity data all the same, just in case.  :)

Cheers!
